aws/userdata/update-repos.sh

```diff
@@ -26,6 +26,23 @@ export VERSION
 export PACKAGING_BRANCH
 
 if [ -z "$DOCKER_ONLY" ]; then
+  INSTANCE_ID=$(curl --retry 5 http://169.254.169.254/latest/meta-data/instance-id)
+
+  # We keep a shared XFS volume as the source of truth to avoid having to
+  # redownload from S3 every day, which can take a *really* long time.
+  VOLUME_ID="vol-096d2c45d13d3a865"
+
+  # Note: We can use try_really_hard from worker.sh because this is source'd, not
+  # executed.
+  try_really_hard aws ec2 attach-volume \
```

Comment:
I think this needs to check if it's already mounted: it can be called multiple times on the same worker if we're building multiple versions at the same time, e.g. for security updates

Reply:
Added the mount test